README.md

```diff
@@ -102,15 +100,17 @@ Now, the contents of the `bindings/LDK` folder will have been completely regener
 ### Configuring Xcode for Framework Compilation
 Go to the `xcode` folder and open `LDKFramework.xcworkspace`.
 
-Then, navigate to the "LDKFramework" project and click on the LDKFramework target:
+Then, navigate to the "LDKFramework" project and click on the LDKFramework project target:
 ![](https://user-images.githubusercontent.com/5944973/175575527-97073a18-76fc-4ab0-928f-d40ac643e607.png)
 
-Search for "LDK_DIRECTORY":
-![](https://user-images.githubusercontent.com/5944973/175575621-38224096-4baa-44cc-8345-ec2b871fcbe6.png)
+Search for `LDK_C_BINDINGS_BASE`:
+![](https://user-images.githubusercontent.com/5944973/177420117-bcf7fc32-b07d-440f-84d1-fe2d373c355b.png)
+
+Here, enter the path: `${PROJECT_DIR}/../../bindings/artifacts/ldk-c-bindings`
 
-Here, enter the absolute path you to ldk-c-bindings you saved above.
+Repeat this for `LDK_C_BINDINGS_BINARY_DIRECTORY`, and enter the following value: `${PROJECT_DIR}/../../bindings/artifacts/bin`
 
-Now, do the same for the "LDKFramework_Mac" target.
+DISCLAIMER: If you know what you're doing, you could have cloned `rust-lightning` and `ldk-c-bindings` above to a different directory. This step essentially tells Xcode where exactly they live on your machine. We decided to use `bindings/artifacts` as the suggested default, since it mirrors what we do in CI, and is guaranteed to work as long as builds are green.
```

Comment:
Ehh, I think cloning one full repository inside another one really shouldn't be something we ought to recommend developers do on their machines, especially if they intend to contribute changes. Not to mention it will confuse the git roots.

Reply:
I personally am not sure I am a fan of cloning outside the bounds of the working environment I am in (`ldk-swift`). `bindings/artifacts` is in the `gitignore`.